WTWSMS/decisions/realm_decisions.txt

```diff
@@ -2028,26 +2061,6 @@ decisions = {
 					value = off
 				}
 			}
-			
-			OR = {
-				ai = no
-				AND = {
-					trait = proud
-					prestige = 5000
-				}
-				AND = {
-					trait = ambitious
-					prestige = 5000
-				}
-				AND = {
-					realm_size = 70
-					any_demesne_title = {
-						count = 4
-						tier = DUKE
-					}
-				}
-				prestige = 15000
-			}
```

Comment:
In this Mod, almost every starting kingdom and duchy is titular. Titles were constantly in flux. Besides, YOU are the one who dogs about "dynamic de jure" and "dynamic realms", but the code _that is actually does it_ is restricted beyond any sane reason.

Removing the prohibition of the AI from creating custom realms (and that is precisely what it is, as those numbers are flat-out insane) doesn't suddenly make it common! They still have to have the costs, they still have to have the clay or the duchies/kingdoms.

I am willing to raise the amount of kingdoms needed for Empires, but the prestige requirements are insane. As high as they are now, you might as well put "AI = NO".